components/global/SiteHeader.vue

```diff
@@ -1,40 +1,35 @@
 <template>
-  <header class="container mx-auto flex justify-between">
-    <div class="w-28">
+  <header class="container mx-auto flex items-center justify-between height-limit">
+    <div class="w-28 bg-primary-light rounded-br-md p-3">
       <BaseLink href="/" :inert="isMobileMenuOpen">
         <BaseImage v-if="logo" :src="logo.filename" :alt="logo.alt" />
       </BaseLink>
     </div>
 
+    <!-- this style may have to go back on the ul class after this comment: space-x-6 -->
+
     <nav>
-      <ul class="desktop hidden md:flex space-x-6">
-        <li v-for="menu in nav" :key="menu.id">
+      <ul class="desktop hidden lg:flex mr-2">
+        <li
+          v-for="menu in nav"
+          :key="menu.id"
+          class="text-black opacity-75 flex justify-center content-center w-28 border-t-2 hover:text-primary-light hover:opacity-100 hover:font-semibold hover:border-accent hover:border-t-2 text-sm"
+        >
```

Comment:
Great work on these styles overall!! Just a few things.

- drop the `w-28`. this hardcoded width will be troublesome if the client ever added a menu item with a long name
- Let's simplify the hover state to the following styles: `hover:border-accent` & `hover:opacity-100`. This will be easier to manage and give some distinction between hover and active link states.
- Let's just make the default font weight `font-semibold` and then _not_ change it for any hover or active states. The weight change on hover causes some jumpiness on the screen and we want to mitigate that as much as possible.
- Great job on the top border! Let's add `border-transparent` so we don't have that weird gray bar above the menu items

Reply:
Word. I get what you mean about the hard coded menu item width. I did the fixed width to prevent the items from shifting when changing to bold on hover.